src/Elastic.ApiExplorer/Endpoints/EndpointView.cshtml

```diff
@@ -13,7 +13,9 @@
 		Previous = null,
 		Next = null,
 		NavigationHtml = Model.NavigationHtml,
-		LegacyPage = null,
+		LegacyPages = [],
```

Comment:
This doesn't have to leak into the main site layout after 

https://github.com/elastic/docs-builder/pull/1357/files

is merged. 

Let's merge this one in first and then i'll handle the merge conflicts.

Reply:
https://github.com/elastic/docs-builder/pull/1361/commits/b999745947f3770eda6d54b18042c68fe5d79e53 and https://github.com/elastic/docs-builder/pull/1361/commits/5df98f2a817a0e28d593e419eb681ca0a4f71d3f